.github/ISSUE_TEMPLATE/--content-proposal-.md

```diff
@@ -1,27 +1,29 @@
 ---
-name: "\U0001F4DDContent Proposal "
-about: 'I want to contribute content to amp.dev. '
+name: "\U0001F4DD Content Proposal"
+about: 'I want to contribute content to amp.dev.'
 title: 'Content proposal '
 labels: ''
 assignees: ''
-
 ---
 
-A short description of your contribution:
+## 📝 Content Proposal
+
+### Synopsis
+A clear and concise description of the content you will be contributing.
 
-Type of contribution:
-[ ] Guide
-[ ] Tutorial
-[ ] Example
-[ ] Translation 
-[ ] Other (please specify)
+### Content type
+- [ ] Guide
+- [ ] Tutorial
+- [ ] Example
+- [ ] Translation
+- [ ] Other (please specify)
```

Comment:
This may be problematic for people who do not know how to use Markdown task lists. We never tell them to place the x in the box to make a selection.

Reply:
I'd approve further instructions if you wouldn't mind adding them!